### PR TITLE
ui improvements [issue mozilla/parsys#15]

### PR DESCRIPTION
**Pull request by LeoMcA**

### CODE REVIEW

File: assets/javascripts/discourse/templates/modal/login-prompt.hbs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{{#d-modal-body title="expose_emails_in.login_prompt.title" class="login-prompt"}}
+  <div>
+    <h3>
+      {{#if model.stranger_danger}}
+        {{i18n "expose_emails_in.login_prompt.body_stranger"}}
+      {{else}}
+        {{i18n "expose_emails_in.login_prompt.body"}}
+      {{/if}}
+      {{d-button  class="btn-primary login-button"
```
Comment (comzeradd) [inline assets/javascripts/discourse/templates/modal/login-prompt.hbs:9]: I don't know if this is some sort of standard code style for handlebars, but you seem to have two spaces here.

File: assets/javascripts/discourse/templates/modal/login-prompt.hbs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                  label="expose_emails_in.log_in_sign_up"}}
+    </h3>
+    {{#if model.category.email_in}}
+      <div class="email-in">
+        {{#if model.stranger_danger}}
+          {{i18n "expose_emails_in.login_prompt.email_stranger"}}
+        {{else}}
+          {{i18n "expose_emails_in.login_prompt.email"}}
+        {{/if}}
+        <a  href="{{model.mailto}}"
```
Comment (comzeradd) [inline assets/javascripts/discourse/templates/modal/login-prompt.hbs:21]: Same here.

**Review by comzeradd — APPROVED** [review]
Code looks good 👍

File: assets/javascripts/discourse/templates/modal/login-prompt.hbs (hunk 1 above)
Comment (LeoMcA) [reply assets/javascripts/discourse/templates/modal/login-prompt.hbs:9]: Yeah, it's just to make all the attributes line up (the extra space is to make it tab-aligned) - not sure if it's a standard, but I think I've seen it used elsewhere

**Pull request merged**